### ISSUE TITLE
timezone issue

### ISSUE THREAD

**Issue by on99** [metabase/metabase#8328]
### Bugs
From date picker I set **on 2018-08-15** and get some metric grouped by datetime hour, but metrics from **2018-08-14** is returned. 

- Your databases: MySQL 5.7 (timezone CST which stands for China Standard Time)
- Metabase version: v0.29.3 (report timezone Asia/Hong_Kong)
- Metabase hosting environment: Docker (JAVA_TIMEZONE=Asia/Hong_Kong)
- Metabase internal database: MySQL 5.7

![image](https://user-images.githubusercontent.com/13196393/44139668-56d66934-a0aa-11e8-89c1-52124243f155.png)

**Comment by on99** [comment]
@senior It does not raise any exception when I connect to MySQL and execute the command in Sequel Pro

**Comment by on99** [comment]
I found that date picker `current day` works well but `before` `after` `on` does not, btw, I have not tested the others.

**Comment by senior** [comment]
I have been able to reproduce this issue and found it to be related to how we are associating date parameters with our MySQL queries. The problem is the same as what https://github.com/metabase/metabase/issues/8262 hit for native queries, just hit via the query builder.

I just merged https://github.com/metabase/metabase/pull/8353 to fix the issue. The fix will be included in the 0.30.1 release which should be out soon.
